apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/avstemming/regulering/VerifiserReguleringssummer.kt

```diff
@@ -0,0 +1,97 @@
+package no.nav.etterlatte.utbetaling.avstemming.regulering
+
+import no.nav.etterlatte.libs.common.retryOgPakkUt
+import no.nav.etterlatte.libs.common.vedtak.Utbetalingsperiode
+import no.nav.etterlatte.libs.common.vedtak.VedtakDto
+import no.nav.etterlatte.libs.common.vedtak.VedtakInnholdDto
+import no.nav.etterlatte.libs.ktor.token.Systembruker
+import no.nav.etterlatte.utbetaling.VedtaksvurderingKlient
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Utbetaling
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.UtbetalingDao
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Utbetalingslinje
+import org.slf4j.LoggerFactory
+import java.math.BigDecimal
+import java.time.YearMonth
+
+class VerifiserReguleringssummer(
```

Comment:
Er dette egentlig kun for regulering? Kunne det gitt mening å hatt denne gående fast for å kunne oppdage eventuelle avvik?

Reply:
Einig, denne bør få eit meir generelt namn. Tenkjer han kan køyre på alle vedtak.